.changeset/fifty-rocks-camp.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/next-sample": minor
+"@logto/next": minor
+"@logto/node": minor
```

Comment:
Should we mark these as a major change? Will it break things if they use `^` with the version number in package.json?

Reply:
The modifications will not impact the utilization, as compatibility is maintained.